purchases/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,13 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools">
+    <!-- Ignoring errors in API <21 since mockk-android is only supported starting in API 21 -->
+    <uses-sdk tools:overrideLibrary="io.mockk,io.mockk.proxy.android"/>
```

Comment:
Need this since mockk-android is only android api 21+. If we try to run integration test in less than that it would fail at runtime. Before, it wouldn't even run (since the integration test flavor was 21+). But I think this is ok?